doxia-modules/doxia-module-markdown/src/test/java/org/apache/maven/doxia/module/markdown/MarkdownParserTest.java

```diff
@@ -361,7 +362,7 @@ protected String parseFileToHtml( String file ) throws ParseException, IOExcepti
         }
     }
 
-    /** @throws Exception  */
+    /** @throws Exception if the document parsing failed. */
```

Comment:
no  period, or just delete this one. We don't need doc comments for tests

Reply:
Deleted